src/Manager.js

```diff
@@ -14,24 +14,30 @@ const convertBooleanToSensorState = bool => bool ? 'ON' : 'OFF';
 export default class Manager {
   constructor() {
     this.logger = logger.child({ name: 'Manager' });
+    this.config = new _config(() => {
+      this.initializeOnvifDevicesFunctions();
+    });
 
     this.init();
   }
 
   init = async () => {
     this.logger.info('Beginning initialization...');
 
-    this.publisher = new MqttPublisher(config.get('mqtt'));
+    this.publisher = new MqttPublisher(this.config.get('mqtt'));
     await this.publisher.connect();
     await this.publisher.publish_service_status('ON');
 
-    this.subscriber = new OnvifSubscriberGroup();
-
-    this.initializeOnvifDevices(config.get('onvif'));
-    this.subscriber.withCallback(CALLBACK_TYPES.motion, this.onMotionDetected);
+    this.initializeOnvifDevicesFunctions();
 
     this.onExitSendStatus();
-};
+  };
+
+  initializeOnvifDevicesFunctions = () => {
+    this.subscriber = new OnvifSubscriberGroup();
```

Comment:
I think we may need to add some teardown / deconstructor logic to remove old event subscribers if an OnvifSubscriberGroup already exists to avoid duplicate events between garbage collection cycles.